js/authentication.md

```diff
@@ -1366,6 +1369,49 @@ class CustomButton extends React.Component {
 export default CustomButton;
 ```
 
+#### Launching the Hosted UI in React Native 
+
+With React Native, you can use `withOAuth` HOC to launch the hosted UI experience. Just wrap your app's main component with our HOC:
+
+> ***The Hosted UI support is only available for React/ React Native / Web***
+
+```javascript
+import { withOAuth } from 'aws-amplify-react-native';
+
+class App extends React.Component {
+  render() {
+    const {
+      oAuthUser: user,
+      oAuthError: error,
+      hostedUISignIn,
+      facebookSignIn,
+      googleSignIn,
+      amazonSignIn,
+      customProviderSignIn,
+      signOut,
+    } = this.props;
+
+    return (
+      <SafeAreaView style={styles.safeArea}>
+        {user && <Button title="Sign Out" onPress={signOut} icon='logout' />}
+        <ScrollView contentContainerStyle={styles.scrollViewContainer}>
+          <Text>{JSON.stringify({ user, error, }, null, 2)}</Text>
+          {!user && <React.Fragment>
+            <Button title="Cognito" onPress={hostedUISignIn} />
```

Comment:
Let's break this into two pieces:
1. Show how to launch the default hosted UI with `hostedUISignIn`
2. Show how you can skip the hosted UI and redirect straight to one of the social providers. Explain that this is for building a custom UI with buttons if you do not want to show the Cognito UI however it will still create a User Pool entry once the OAuth redirect has completed.

Reply:
Addressed